Specs/Types/Date.js

```diff
@@ -424,6 +424,9 @@ describe('Date', function(){
 		it('should return the minutes as two digits', function(){
 			expect(d.format('%M')).toEqual('02');
 		});
+		it('should return the month as one digit', function(){
+			expect(d.format('%n')).toEqual('11');
```

Comment:
This test is not really suitable to test `%n`, is it? You should pick a month < 10 and make sure there is no leading `0`.